src/apps/dashboard/routes/users/access.tsx

```diff
@@ -307,7 +309,7 @@ const UserLibraryAccess: FunctionComponent = () => {
                                 key={Item.Id}
                                 className='chkDevice'
                                 itemId={Item.Id}
-                                itemName={Item.Name}
+                                itemName={Item.CustomName || Item.Name}
```

Comment:
This may cause a problem if a user choose to use an empty string as a custom name (as silly as that might be). Wondering if this should be flipped to use nullish coalescing (??) instead of | |

Reply:
I think we don't need empty custom name. `||` will work as needed - name will be used.
The user can still use a space (`' '`) though. 😅

Reply:
Fair point, but if we dont need custom name the author should remove it?

Reply:
When a user wants to remove a custom name from a device, they'll clear the textbox and hit save. This does result in an empty custom name, but it's most likely not what the user wants to see.